src/sinks/elasticsearch.rs

```diff
@@ -262,7 +262,7 @@ fn es(
                     // to play games here
                     let body = request.payload.take().unwrap();
                     match body {
-                        SignedRequestPayload::Buffer(body) => builder.body(body).unwrap(),
+                        SignedRequestPayload::Buffer(body) => builder.body(body.to_vec()).unwrap(),
```

Comment:
we should apply that same trick above here.

Reply:
Spoke too soon, on it :)

Reply:
It doesn't appear that the same trick works here.